client/errors.go

```diff
@@ -30,10 +30,24 @@ func classifyError(err error, fallbackType diag.Type, accounts []Account, opts .
 						diag.WithType(diag.ACCESS),
 						diag.WithSeverity(diag.WARNING),
 						ParseSummaryMessage(err),
-						diag.WithDetails("%s", errorCodeDescriptions[ae.ErrorCode()]),
+						diag.WithDetails(errorCodeDescriptions[ae.ErrorCode()]),
```

Comment:
This is incorrect: if the description contains a `%` char it will end up being formatted as an invalid message.